### ISSUE TITLE
[oio-event-agent] Errors not clear during exhume

### ISSUE THREAD

**Issue by sebastienlapierre** [open-io/oio-sds#1009]
If all services are scored to zero and if we try to exhume events like :
```
{"event":"storage.container.state","when":1494855780460546,"url":{"ns":"SCALAIR","account":"scalair_account","user":"1MB_1174"},"data":{"policy":null,"ctime":1490778022602470,"bytes-count":100000000,"object-count":100}}
```
We fall on this error in oio-event-agent:
```
2017-05-22T14:07:15.130810+02:00 sasopenio01 OIO,SCALAIR,oio-event-agent,0: err  1725 1BFCC30 log ERROR handling event 224331094 (bury)
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/oio/event/consumer.py", line 208, in handle
    self.process_event(job_id, event, beanstalk)
  File "/usr/lib/python2.7/site-packages/oio/event/consumer.py", line 231, in process_event
    handler(event, cb)
  File "/usr/lib/python2.7/site-packages/oio/event/filters/base.py", line 19, in __call__
    self.process(env, cb)
  File "/usr/lib/python2.7/site-packages/oio/event/filters/volume_index.py", line 58, in process
    volume_id, container_id, content_id, chunk_id, args)
  File "/usr/lib/python2.7/site-packages/oio/event/filters/volume_index.py", line 35, in _chunk_push
    volume_id, container_id, content_id, chunk_id, **args)
  File "/usr/lib/python2.7/site-packages/oio/rdir/client.py", line 191, in chunk_push
    json=body)
  File "/usr/lib/python2.7/site-packages/oio/rdir/client.py", line 173, in _rdir_request
    resp, body = self._direct_request(method, uri, params=params, **kwargs)
  File "/usr/lib/python2.7/site-packages/oio/api/base.py", line 93, in _direct_request
    resp = session.request(method, url, **out_kwargs)
  File "/usr/lib/python2.7/site-packages/requests/sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/site-packages/requests/sessions.py", line 596, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/site-packages/requests/adapters.py", line 473, in send
    raise ConnectionError(err, request=request)
ConnectionError: ('Connection aborted.', BadStatusLine("''",))
```

**Comment by jfsmig** [comment]
Using the 4.x series ?

**Comment by sebastienlapierre** [comment]
Yes 4.0.0.a4

**Comment by kamel-rahim** [comment]
It's look like that it's not the event-agent but the rdir the problem.
It happened that the rdir could not withstand heavy load, but it's fix by the PR #1013.
